test/bib-marc-test.js

```diff
@@ -248,6 +248,9 @@ describe('Bib Marc Mapping', function () {
           assert.equal(bib.literal('nypl:extent'), 'v. ;')
 
           assert.equal(bib.literal('dcterms:alternative'), 'Cobbett\'s weekly political register (London, England : 1802)')
+          assert.equal(
+            bib.literal('nypl:titleDisplay'), 'Cobbett\'s weekly political register [electronic resource].'
```

Comment:
With the changes to the test json file, this test will fail if using the old version of `field-mapping-bib.json`